### ISSUE TITLE
Reading newer google sheets

### ISSUE THREAD

**Issue by elinw** [mikesall/charted#44]
I noticed that with Googlesheets you are using format=csv which is the old Sheets API and won't work with new spreadsheets (it still works with ones that were created before the old version of the sheets API was dropped).    This is how I dealt with the problem in R, using some modified code from Andrei de Vries at Revolution
https://github.com/elinw/ShinyGoogleSheets/blob/nodata/scripts/readGoogleSheet.R

Basically the sheet needs to be published to the web (not just shared), and then you access it by appending /pubhtml.
